### PR TITLE
Create basic structure rockets

### PR DESCRIPTION
**Pull request by BonkeGcobo**
1) We added a reducer for the rockets.
2) Added routing to each page.

### CODE REVIEW

File: src/redux/rockets/rockets.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const nameInitialState = [];
```
Comment (ahangarha) [inline src/redux/rockets/rockets.js:1]: I think this variable can be called something like `initialState`

**Review by ahangarha — APPROVED** [review]
# Approved :tada: 

Great job. There is one optional recommendation for variable name. I think it would be good to take it into consideration.

Feel free to merge this PR :rocket:

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,20 @@
...
-    </BrowserRouter>
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="/" element={<Rockets />} />
+        <Route path="/Missions" element={<Missions />} />
+        <Route path="/profile" element={<Profile />} />
+      </Routes>
+    </div>
   );
```
Comment (ahangarha) [inline src/App.js:18]: Wasn't it better to have BrowserRouter there? Becaue all routes are here and it makes more sense to have the BrowserRouter here as well. What do you think?

Comment (BonkeGcobo) [reply src/App.js:18]: I have always thought it should encapsulate the entire App component, but I mean it is good

**Pull request merged**